src/ui/canvas/fb/mxcfb.h

```diff
@@ -18,8 +18,8 @@
  *
  * @ingroup Framebuffer
  */
-#ifndef __ASM_ARCH_MXCFB_H__
-#define __ASM_ARCH_MXCFB_H__
+ 
+#pragma once
```

Comment:
Don't edit third-party sources. This will only increase the diff if we update it from upstream.